src/gsi/directDA_radaruse_mod.f90

```diff
@@ -0,0 +1,514 @@
+MODULE directDA_radaruse_mod
```

Comment:
This is new file, please make sure following GSI code standard, such as adding "r_kind" or "r_single" to numbers used here.

Reply:
We have 5 new files (directDA_radaruse_mod.f90, radarZ_global_module.f90, radarZ_interface.f90, radarZ_module.f90, and read_radarref_mosaic_directDA.f90), and I modified them to meet the code standard poss. Basically, I added kind attributes to INTEGER and REAL declaration and make them similar like other GSI codes. If these new files have issues, I will promptly work to resolve the issue.